### ISSUE TITLE
Deployed contracts management

### ISSUE THREAD

**Issue by shunsukew** [AstarNetwork/swanky-cli#12]
Right now, deployed contracts are managed in `swanky.config.json` file like below. 
```
"contracts": [
    {
      "name": "flipper",
      "address": "5GYsjgJnmXdbNACtYMtMYesFbu3rxt7AK1DCoamxMg1b1DpZ"
    },
    {
      "name": "flipper",
      "address": "5Gw6JUoUDMs6sbukc5YxmPrxNrJDGmMPGKJVP7zJaHAa76oc"
    }
  ]
```
However, as the number of supported networks grows, it gets hard to know which contract is deployed to which network.

Several solutions can be considered.
- Having another file/folder for contract management purpose, and preparing each network's section (`deployments` for e.g. in truffle cli).
- Adding network name in contracts item in `swanky.config.json`.
I think former one is more extendability.

**Comment by codespool** [comment]
Agreed, the contract object needs to be extended.
Let's add the network and the timestamp at least.
What I've been thinking about is what to do when network is local (swanky node) and it gets reset? Should we delete the contract record as well? (or keep it in a separate, temp config?)

**Comment by shunsukew** [comment]
That's true...
Right now, node's default behavior is non-persistent mode. That means every time user `swanky node start`, chain state starts from #0 block.

I think default behavior should be persistent mode, so that users can manage contracts even in local nodes. We provide `--tmp` option to use non-persistent mode. When using this or purge chain state, they should modify config files manually.

**Comment by codespool** [comment]
solved in #30
